packages/big-design/src/components/AccordionGroup/AccordionPanel/AccordionPanel.tsx

```diff
@@ -0,0 +1,63 @@
+import { Text } from '@bigcommerce/big-design';
+import { ExpandLessIcon, ExpandMoreIcon } from '@bigcommerce/big-design-icons';
+import React, { HTMLAttributes, memo } from 'react';
+
+import { useUniqueId } from '../../../hooks';
+import { BoxProps } from '../../Box/Box';
+import { useAccordion } from '../hooks/useAccordion';
+
+import { StyledAccordion, StyledAccordionButton, StyledBox, StyledGridItem } from './styled';
+
+export interface AccordionPanelProps extends HTMLAttributes<HTMLDivElement> {
```

Comment:
Should this actually extend from `PanelProps`? 🤔

Reply:
I kept this as is since the parent is not a `Panel.`

Reply:
Can you provide some clarification on why you suggest this?

Reply:
Side bar from Jorges comment, do we even need consumers to pass in `HTMLDivElement` attributes? If we leave this out right now, we can always add back in when consumers need it. We can't remove it without a breaking change if we add it in now.